src/__tests__/personProcessing.test.ts

```diff
@@ -0,0 +1,186 @@
+import { createPosthogInstance } from './helpers/posthog-instance'
+import { uuidv7 } from '../uuidv7'
+import { logger } from '../utils/logger'
+jest.mock('../utils/logger')
+
+describe('person processing', () => {
+    describe('init', () => {
+        it("should default to 'always' process_person", async () => {
+            // arrange
+            const token = uuidv7()
+
+            // act
+            const posthog = await createPosthogInstance(token, {
+                __preview_process_person: undefined,
+            })
+
+            // assert
+            expect(posthog.config.__preview_process_person).toEqual('always')
+        })
+        it('should read process_person from init config', async () => {
+            // arrange
+            const token = uuidv7()
+
+            // act
+            const posthog = await createPosthogInstance(token, {
+                __preview_process_person: 'never',
+            })
+
+            // assert
+            expect(posthog.config.__preview_process_person).toEqual('never')
+        })
+    })
+
+    describe('identify', () => {
+        it('should fail if process_person is set to never', async () => {
+            // arrange
+            const token = uuidv7()
+            const onCapture = jest.fn()
```

Comment:
nit-picking it looks like we could lift these two setup calls  into a beforeEach to get them out of the test methods

Reply:
Done